models/issue.go

```diff
@@ -525,6 +525,24 @@ type Label struct {
 	IsChecked       bool `xorm:"-"`
 }
 
+func (m *Label) UpdateIssueCounts() error {
```

Comment:
Should this function be removed, or would you like to write anther huge function to recount all labels(then I can plugin that function into admin panel)